### DISCUSSION TITLE
本科毕业论文的版式似乎有误

### DISCUSSION THREAD

**Discussion by Cyrisub** [note286/xduts#41] [Q&A] [answered]
1. 根据《本科生毕业设计（论文）撰写规范》，主体部分应在右面另起一页，但中英文摘要似乎不应计入主体。鉴于这两个部分一般不应超过一页，同时按《撰写规范》的示例来看，中英文摘要恰好在一页的正反两面是比较合理的，并且摘要也不需要页码。
2. 尽管《撰写规范》写明了页面参数，但根据我在 Word 中的测试，《撰写规范》后的示例实际使用的是没有 1 厘米装订线的页面设置……当然示例里还有更多有出入的地方，例如摘要之前的所有页面都没有使用对称页边距（也就是两侧一样宽），在章标题使用了粗体，对短标题更改了字符间距等等。
不过我觉得示例里的布局在显示器上看着比较协调，按《规范》中的布局显得过于失调了。

PS：《撰写规范》看似写了一大堆格式，实际上根本无法用上面给出的参数调到示例的模样，最重要的段前段后间距也没有规定……因此提出这两点仅供参考吧。

**Comment by note286** [comment, marked as answer]
1. 中英文摘要均在奇数页开始有两点考虑，一点是惯例，第二点是会有人摘要写的很长，超过一页。

2. 论文最后是要装订的，所以要留有装订边距，在XDUTS中是提供了接口可以调整为对称页边距的。

PS: 你甚至没有star。

**Reply by Cyrisub** [reply]
> 1. 中英文摘要均在奇数页开始有两点考虑，一点是惯例，第二点是会有人摘要写的很长，超过一页。

一页摘要大概在 800~900 字左右，按《规范》要求在 300 字左右为宜，个人认为超过一页的摘要出现在一篇**本科**毕业论文的必要性并不高，甚至这么长的摘要是应该避免的。

> 2. 论文最后是要装订的，所以要留有装订边距，在XDUTS中是提供了接口可以调整为对称页边距的。

按《规范》内侧 3 厘米外侧 2 厘米的要求，如果把装订线的 1 厘米计入在内侧边距中，装订完的页心正好在中间。所以我并不确定这是《规范》的笔误还是确实的有意为之，毕竟 6 厘米的总边距真的很浪费纸……

> PS: 你甚至没有star。

对不起 QAQ，已经 star。

**Reply by note286** [reply]
你还要考虑页眉呢，中英文摘要不能在同一张纸的正反面。

**Reply by note286** [reply]
本科生的就这样，行间距也大，一眼看去，一页没多少内容。
